package/MDAnalysis/analysis/pca.py

```diff
@@ -0,0 +1,353 @@
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- http://www.MDAnalysis.org
+# Copyright (c) 2006-2015 Naveen Michaud-Agrawal, Elizabeth J. Denning, Oliver Beckstein
+# and contributors (see AUTHORS for the full list)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+"""
+Principal Component Analysis (PCA) --- :mod:`MDAnalysis.analysis.pca`
+=====================================================================
+
+:Authors: John Detlefs
+:Year: 2016
+:Copyright: GNU Public License v3
+
+This module contains the linear dimensions reduction method Principal Component
+Analysis (PCA). PCA sorts a simulation into 3N directions of descending
+variance, with N being the number of atoms. These directions are called
+the principal components. The dimensions to be analyzed are reduced by only
+looking at a few projections of the first principal components. To run a
+Principal Component Analysis, please refer to the :ref:`PCA-tutorial`.
+
+This module constructs a covariance matrix wherein each element of the matrix
+is denoted by (i,j) row-column coordinates. The (i,j) coordinate is the
+influence of the of the ith coordinates on the jth coordinates
+of a given trajectory. The eponymous components are the
+eigenvectors of this matrix.
+
+For each eigenvector, its eigenvalue is the variance that the eigenvector
+explains. Stored in :attribute:`cumulated_variance`, a ratio for each number of
+eigenvectors up to index i is provided to quickly find out how many principal
+components are needed to explain the amount of variance reflected by those i
+eigenvectors. For most data, :attribute:`cumulated_variance` will be
+approximately equal to one for some n that is significantly smaller than the
+total number of components, these are the components of interest given
+by Principal Component Analysis.
+
+From here, we can project a trajectory onto these principal components and
+attempt to retrieve some structure from our high dimensional data.
+
+For a basic introduction to the module, the :ref:`PCA-tutorial` shows how
+to perform Principal Component Analysis.
+
+.. _PCA-tutorial:
+The example uses files provided as part of the MDAnalysis test suite
+(in the variables :data:`~MDAnalysis.tests.datafiles.PSF` and
+:data:`~MDAnalysis.tests.datafiles.DCD`). This tutorial shows how to use the
+PCA class.
+
+First load all modules and test data ::
+    >>> import MDAnalysis as mda
+    >>> import MDAnalysis.analysis.pca as pca
+    >>> from MDAnalysis.tests.datafiles import PSF, DCD
+
+Given a universe containing trajectory data we can perform PCA using
+:class:`PCA`:: and retrieve the principal components.
```

Comment:
the last `::` are not necessary. And please mention in the test that `PCA` is also a class. In the rendered docs this reads `to perform PCA we can use PCA`. something like `to perform a PCA we can use the PCA class` is better.